recipes/rotor/all/conanfile.py

```diff
@@ -32,6 +33,7 @@ class RotorConan(ConanFile):
         "enable_thread": False,
         "multithreading": True,
         "enable_ev": False,
+        "enable_fltk": False,
```

Comment:
This option should be removed in the `config_options` method below as `enable_ev` is, when the version is less than 0.30 so that it does not affect the package if it's not valid